websites/D/Discohook/presence.ts

```diff
@@ -0,0 +1,22 @@
+const presence = new Presence({
+		clientId: "1036322932932218880",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
```

Comment:
Imgur links are now preferred over Discord asset names

Reply:
Switched from asset to Imgur URL

Reply:
Make sure to mark these as resolved when you've completed them.